### PR TITLE
Edit clubs/README.md

### LINKED ISSUE #722: Make it clearer how to use the template slideshow presentations
We have a few template slideshow presentations for meetings in https://drive.google.com/folderview?id=0B34YNYNQ6z3YWWxHRGdnUVVpbzA&usp=sharing&tid=0B34YNYNQ6z3YbE1JVmZfdW9Qc3c that are linked from the club guide (https://github.com/hackclub/hackclub/blob/master/clubs/README.md).

They're meant to be used as follows:

1. Club leader makes a copy of the appropriate slideshow for their meeting (`File > Make a copy...`)
2. Club leader replaces all text in brackets (ex. `[HIGH_SCHOOL]`) with appropriate text for their club and makes any other changes they want

Right now the links to the slideshows aren't super prominent and we've had a couple clubs get confused by the guide on using them.

### PR DESCRIPTION
**Pull request by jkwok91**
**Under discussion; do not merge**

@zachlatta: please review changes and suggest more changes.

Closes #722.

### CODE REVIEW

**Comment by jkwok91** [comment]
Most importantly, the first section needs some help. A more compelling definition of "Hack Club" would be good.

File: clubs/README.md [hunk 1]
```diff
@@ -193,38 +188,35 @@ apps, websites, and games).
...
 
-We recommend the format below for your first meeting. For each item in the
-schedule, there is a corresponding (and linked) section. We've also created a
-slideshow presentation to help you run your first meeting.
-[Click here][presentation] to go to it. Make a copy of the presentation to
-customize it for your club (to do this, go to `File > Make a copy...`). We'll
-refer to this presentation throughout the sections below.
+We recommend the following format for your first meeting. There are
+corresponding slides in the [Workshop Day presentation][workshop_day_template],
+and we've gone into more detail in the linked sections below.
```
Comment (zachlatta) [inline clubs/README.md]: The slides for the first meeting are different than the slides for the workshop day. Does this remove references to the first meeting slides entirely?

File: clubs/README.md [hunk 2]
```diff
@@ -280,53 +272,50 @@ The name game is a simple icebreaker activity that will warm the atmosphere and
...
-3. Say that after the Personal Website workshop, the rest of the meeting will be
-   completely free-form for them to continue building their websites. Show the
-   slides with examples of websites other people in Hack Clubs have made to give
-   your members ideas for directions they can take their sites. Say that at the
-   end of the next meeting, everyone is going to come to the front and demo what
-   they built.
+   will take around 45 minutes.
+3. Explain that after the workshop, the rest of the meeting will be completely
+   free-form for members to continue building their websites. Show examples of
+   websites that other Hack Clubs' members have    made to give your members
```
Comment (zachlatta) [inline clubs/README.md]: Extra spaces

File: clubs/README.md [hunk 3]
```diff
@@ -398,53 +388,90 @@ use it. [Click here][demo] for an example of what a project demo looks like
...
+   for what you'll be doing.
+3. Customize the remainder of the slides according to the type of meeting:
+   - For Workshop Day:
+     - Choose the workshop you'll want to run for this meeting and put its name
+       in the appropriate slide.
+     - Add the images corresponding to this workshop. A good tip is to add one
+       or two standard examples of what the workshop will produce, and then one
+       or two examples of what it would look like to expand on the
+       workshop, as inspiration.
+     - Include a link at the end to the workshop on `hackclub.com/[WORKSHOP]`.
```
Comment (zachlatta) [inline clubs/README.md]: `workshops.hackclub.com/[workshop]`

File: clubs/README.md [hunk 4]
```diff
@@ -508,10 +532,10 @@ music back on (not too loud, people should feel comfortable talking).
...
 #### Next Meeting Reminder
 
-Five minutes before the meeting is over, get everyone's attention to remind them
-about the next meeting. Say when and where it'll be and that everyone will be
-finishing up what they worked on today and demoing them at the end of the
-meeting.
+Five minutes before the meeting is over, get everyone's attention to remind
+them about the next meeting. Say when and where it'll be and that everyone will
+be finishing up what they worked on today and demoing at the end of the
+meeting.
```
Comment (zachlatta) [inline clubs/README.md]: Extra spaces at end of line.

**Comment by zachlatta** [comment]
For the most part, looks good. I left a couple comments.

Couple things:

- Make sure to remove extra whitespace at the ends of lines
- The slideshow templates for the first and second meeting are different than the workshop day and hack day templates. Can you make sure they're included?

**Comment by zachlatta** [comment]
Couple minor things, but I figure we can fix them later :ok_hand:.

**Pull request merged**